ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
@jamienoss do you mean swap if (!item.queryPersistRefresh()) with  if (savedEclCRC != eclCRC) conditional?  If I do that and CRC is missing, then the file will be re-built - I'm not sure that is what is being asked for.  @jakesmith please can you comment on this?